lib/new_relic/agent/instrumentation/logstasher/instrumentation.rb

```diff
@@ -0,0 +1,24 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module NewRelic::Agent::Instrumentation
+  module LogStasher
+    INSTRUMENTATION_NAME = 'LogStasher'
```

Comment:
```suggestion
    INSTRUMENTATION_NAME = NewRelic::Agent.base_name(name)
```

For instrumentation where the module name is an exact match, you can send `name` over to `base_name`. The intention behind doing so was to keep things dynamic and permit modules to be renamed in future, but granted we don't anticipate doing so often.